uiowa/uiowa
# I2555 srop view

Closes #2555 


# How to test
1. blt ds --site=grad.uiowa.edu
2. blt frontend
3. Add a few 'people' with the 'Scholar' person type selected and multiple srop sessions selected (some similar, some unique to each individual)
4. go to grad.local.drupal.uiowa.edu/past-scholars
4a. Make sure there's a linked list of the srop sessions that were selected for the newly created people.
4b. Click on list and see that it displays a summary of users that have selected that current session (ex: 2020 (2) if two people have selected the year 2020)
4c. Make sure the 2020 session year URL path displays as: grad.local.drupal.uiowa.edu/dei/srop/past-scholars/2020
